pkg/cel/environment.go

```diff
@@ -57,6 +57,7 @@ func DefaultEnvironment(options ...EnvOption) (*cel.Env, error) {
 		// default stdlibs
 		ext.Lists(),
 		ext.Strings(),
+		cel.StdLib(),
```

Comment:
It doesn't hurt to put this here, but isn't this implied by `cel.NewEnv`?

https://github.com/google/cel-go/blob/master/cel/env.go#L926